app/build.gradle

```diff
@@ -42,7 +43,7 @@ android {
 	    sourceSets {
                 main {
                     manifest.srcFile 'AndroidManifest.xml'
-		    java.srcDirs = ['src']
+		            java.srcDirs = ['src']
```

Comment:
looks like the indentation wasn't quite right in the first place, with tabs instead of spaces, if you want to fix this in this commit can you switch to using spaces to keep alignment consistent?